docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -84,15 +84,15 @@ then add the relevant configuration properties in `{config-file}`.
 [source,properties]
 .Example `{config-file}`
 ----
-# datasource configuration
-quarkus.datasource.db-kind = postgresql
+quarkus.datasource.db-kind = postgresql <1>
 quarkus.datasource.username = hibernate
 quarkus.datasource.password = hibernate
 quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/hibernate_db
 
-# drop and create the database at startup (use `update` to only update the schema)
-quarkus.hibernate-orm.database.generation=drop-and-create
+quarkus.hibernate-orm.database.generation=drop-and-create <2>
 ----
+<1> xref:datasources[Configure the datasource].
+<2> Drop and create the database at startup (use `update` to only update the schema).
```

Comment:
Maybe a silly question ... shouldn't we be hinting users towards flyway/liquibase for schema management and suggesting `quarkus.hibernate-orm.database.generation` more for "testing purposes"? 🙈🙈🙈

Reply:
Good idea, PR welcome! :)